tests/unknown-modules.txt

```diff
@@ -25,6 +25,6 @@ We print warnings when we cannot find a module/package
 
     >>> graph = ModuleGraph()
     >>> graph.path.append('.')
-    >>> graph.parseFile('marmalade.py')
+    >>> graph.parseFile('marmalade.py', False)
```

Comment:
Style tip: always use keyword-argument syntax when passing boolean literals, so one doesn't have to look up the method definition to figure out what False means here.